main/src/views/stats/chart2.vue

```diff
@@ -1,7 +1,7 @@
 <template>
-  <div>
-    <h2 class="number">{{ total }}</h2>
-    <p class="description">Dossiers accompagnés avec DossierFacile</p>
+  <div class="flex-reverse">
```

Comment:
C'était quoi le but ? Ça rajoute de la complexité, est-ce que c'est utile ?

Reply:
En fait, le contenu doit être après le titre, ici si on garde tel quel, le total devient le titre.
"Headings should describe the content that follows it"
-> Une alternative pourrait etre de considérer qu'il ne s'agit pas de titre.

Reply:
Est-ce que c'est sûr que c'est un hack valide ? C'est pour les lecteurs d'écrans et ils ne prennent pas en compte le css ?
Parce que en parlant de css et d'accessibilité, il faut aussi prendre en compte la possibilité de supprimer complètement le css et dans ce cas ça se retrouverait à l'envers… Ici ça change pas grand chose donc je propose de merger comme ça mais de garder en tête que ça devrait pas être une solution "générique" si un autre cas dans le même genre arrive